integrity-backend/input-proofmode-signal-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-recorder.json

```diff
@@ -0,0 +1,50 @@
+{
+  "recorderMetadata": [
+    {
+      "service": "api",
+      "info": [
+        {
+          "type": "external",
+          "values":
+          {
+            "name": "proofmode-1.0.0"
```

Comment:
@YurkoWasHere we'll try to find this info in the zip?

Reply:
Currently does not exist, asked for it but who knows

Reply:
Let's leave it here for now. Hopefully we can fill it in future.